papaya-web/src/main/java/pl/edu/agh/papaya/rest/projects/service/SprintsRestService.java

```diff
@@ -148,23 +156,30 @@
             );
 
             @SuppressWarnings({"MultipleStringLiterals"})
-            Duration timePlanned = Duration.ofMinutes(
-                    AssertionUtil.requireNonNegative("timePlanned", sprintDto.getTimePlanned())
+            Duration finalTimePlanned = Duration.ofMinutes(
+                    AssertionUtil.requireNonNegative("finalTimePlanned", sprintDto.getFinalTimePlanned())
```

Comment:
Te wartości chyba tez już nie są tutaj wymagane - chociaż trzeba będzie sprawdzić, że zostały zaciągnięte z Jiry, ale to w innym zadaniu

Reply:
Jak już zostawiasz sprawdzanie `timeBurned` i `finalTimePlanned`, to to trzecie z Jiry też może tutaj być

Reply:
to to trzecie się doda w pr z jirą